vector/src/main/java/im/vector/app/features/home/UnreadMessagesSharedViewModel.kt

```diff
@@ -144,7 +144,7 @@ class UnreadMessagesSharedViewModel @AssistedInject constructor(
                         this.memberships = listOf(Membership.JOIN)
                         this.spaceFilter = SpaceFilter.OrphanRooms.takeIf {
                             !vectorPreferences.prefSpacesShowAllRoomInHome()
-                        }
+                        } ?: SpaceFilter.NoFilter
```

Comment:
In this case I think a if/else would be clearer:
```kotlin
if (vectorPreferences.prefSpacesShowAllRoomInHome()) {
    SpaceFilter.NoFilter
} else {
    SpaceFilter.OrphanRooms
}
```

Reply:
agreed, will update :+1:

Reply:
https://github.com/vector-im/element-android/pull/6666/commits/09d840506e8b3e8e074608b7c443778b90120c77